ext/native-decls/CancelEvent.md

```diff
@@ -10,3 +10,9 @@ void CANCEL_EVENT();
 
 Cancels the currently executing event.
 
+## Examples
+```lua
+    AddEventHandler('ptFxEvent', function(sender, eventName, eventData)
```

Comment:
`AddEventHandler` is still incorrectly indented, remove the spaces to the left so it's properly indented. This is how it should look:

```lua
AddEventHandler('ptFxEvent', function(sender, eventName, eventData)
	CancelEvent()
end)
``` 

Code to copy:

`````
```lua
AddEventHandler('ptFxEvent', function(sender, eventName, eventData)
	CancelEvent()
end)
``` 
`````

Maybe just replace your `lua` block with what I just sent you 😀